### ISSUE TITLE
Feature Request: UI/UX - WCAG 2 Compliance

### ISSUE THREAD

**Issue by daveedgn** [trinodb/trino#21087]
# Feature Request: UI/UX - WCAG 2 Compliance

## Background

Accessibility is about ensuring that digital technology is usable by people with disabilities. Accessibility is an important aspect of diversity, equity, and inclusion (DEI).

Web Content Accessibility Guidelines (WCAG) 2.0 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content accessible to a wider range of people with disabilities, including blindness and low vision, deafness and hearing loss, learning disabilities, cognitive limitations, limited movement, speech disabilities, photosensitivity and combinations of these. Following these guidelines will also often make your Web content more usable to users in general.

For more information, see: https://www.w3.org/WAI/standards-guidelines/wcag/

## Request

Update the Trino's UIs (e.g. cluster overview, worker list, worker status, query details...) to be compliant with WCAG 2.1 to at least Level AA conformance level.

## Validation

Generate a Google AXE or Lighthouse report showing each page does not have any Critical or Serious/High issues.
